near-sdk-macros/src/core_impl/code_generator/sim_proxy.rs

```diff
@@ -0,0 +1,12 @@
+use proc_macro2::Ident;
+
+pub fn generate_sim_proxy_struct(ident: &Ident) -> proc_macro2::TokenStream {
```

Comment:
Shouldn't this be deprecated or even better removed in the 4.0 release?

Reply:
It will be once sim tests are fully deprecated. Do not want to interrupt current usage patterns